keyvalues/scripts/weapons/mp_weapon_vinson.txt

```diff
@@ -109,7 +109,7 @@ WeaponData
 		"aimassist_adspull_weaponclass"					"precise"
 
 		"damage_near_value"   							"20" // CHANGE/NERF: 30 -> 20 (4 -> 5 bullets)
-		"damage_far_value"								"20"//30
+		"damage_far_value"								"17" // CHANGE/NERF: 20 -> 17 (4 -> 6 bullets)
```

Comment:
20 damage is 5 bullets, so:

```suggestion
		"damage_far_value"								"17" // CHANGE/NERF: 20 -> 17 (5 -> 6 bullets)
```

Reply:
in my defense i had to create that //comment since it was left in vanilla XD
but yes i definitely missed that since i was focused on copying the format that is already available in the near damage comment

Reply:
> in my defense i had to create that //comment since it was left in vanilla XD
Oh right, those can just be removed